google/resource_compute_backend_service.go

```diff
@@ -37,6 +37,32 @@ func resourceComputeBackendService() *schema.Resource {
 				MaxItems: 1,
 			},
 
+			"iap": &schema.Schema{
+				Type: schema.TypeSet,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": &schema.Schema{
+							Type:     schema.TypeBool,
+							Optional: true,
```

Comment:
I wonder whether we should make this Required, that way people don't accidentally try to enable it by putting in the block but forgetting this field

Reply:
I considered this as well. I think about how I work, and commenting out IAP enabled would "disable" it, without having to edit the line and set it as false. What's the standard here?

Reply:
Something else to consider here, should we even have the enabled flag? Should it be implicit by providing the keys?